### ISSUE TITLE
Обработка ответов от сервера

### ISSUE THREAD

**Issue by elagin** [elagin/motobatClient#12]
Например при "create" получаю в CreatePointCallback

> {"error":"Ошибка соединения {\"RESULT\":{\"response\":\"ok\"}}"}
> и
> {"error":"Ошибка соединения {\"ERROR\":{\"text\":\"PREREQUISITES\",\"object\":\"userid\"}}"}

**Comment by rjhdby** [comment]
Я завтра унификацией сетевой части займусь.
Посмотри как в RoleRequest сделано. По факту обработка ошибок вынесена в HTTPClient

**Comment by elagin** [comment]
Да, если ошибки нет, то шелуху нужно очищать в HTTPClient, а в xxxCallback уже только результат отдавать.

**Comment by elagin** [comment]
И вывод сообщений посредствам Toast
